fancy-explosions/BangazonSite
# On my products page shows number of products sold

# Description
On the my products page the page now displays the number sold of each product. Also if user is not logged in the my products nav will not appear on the nav bar. I also fixed the issue of details link not appearing on the ProductType details page.



Fixes Issue # https://github.com/fancy-explosions/BangazonSite/issues/16

## Type of change

Please delete options that are not relevant.



# Testing Instructions
git fetch --all
git checkout cm-issue-16
when logged out make sure that the my products nav does not appear
log back in and buy some products that are on your my products page
then navigate to my products page and make sure the number sold is displaying



# Checklist:
